### ISSUE TITLE
[JAWS + IE11] Slider performance

### ISSUE THREAD

**Issue by phet-steele** [phetsims/john-travoltage#242]
@jessegreenberg, sliders in JAWS + IE11 seem to perform abysmally.

Use this as an example case with the below steps: https://jsfiddle.net/1j1y3s8p/

First problem:
1. Start with the slider at -7
2. Pay attention to the speed of the slider during this step. Hold the right arrow key down until halfway, then release. When holding, you will see the slider tick along relatively slowly. After release, the slider will rapidly move right.

Second problem:
1. Make sure you are now at the max of the slider (also this works at min), hold the right arrow key down for a short time (or left key if you are at min, try 3 seconds). Too long and you may crash JAWS.
2. After releasing from hold, press the left arrow key once. There will be a decent delay until the slider actually decrements to -6.

Hopefully it goes without saying, but both of these issues apply to the leg/arm sliders in this sim.

For phetsims/tasks#831.

**Comment by jessegreenberg** [comment]
@emily-phet the 2 problems in this issue will happen with JAWS/IE11 and *any* HTML slider, as @phet-steele pointed out with the JSFiddle example.  As such, I think this should be marked as ```wontfix```, and we should let Freedom Scientific know about this problem.  Does that sound OK to you?

**Comment by emily-phet** [comment]
@jessegreenberg Sounds good to me.

**Comment by jessegreenberg** [comment]
@lmulhall-phet @JRomero0613, are you still able to reproduce this bug when using JAWS? It impacts sliders and might show up in RIAW.

**Comment by JRomero0613** [comment]
@jessegreenberg I was able to reproduce both issues using JAWS with the fiddle link you gave. I was not able to get it to repeat values that the slider was not on as I was in https://github.com/phetsims/resistance-in-a-wire/issues/155 but it does look like the same issue that is affecting both.

**Comment by jessegreenberg** [comment]
Thanks @JRomero0613, just wanted to make sure. In that  case I will close this issue and add it to the known list.

**Comment by jessegreenberg** [comment]
Wait, @JRomero0613 were you able to reproduce in Firefox or IE11? I just want to make sure I write it down correectly.

**Comment by JRomero0613** [comment]
@jessegreenberg I was able to reproduce in both Firefox and IE11. For some reason the fiddle link would not open in IE11, but I tested the resistance in a wire sliders on IE11 with JAWS and it exhibits the same behavior seen previously.

**Comment by jessegreenberg** [comment]
Thanks @JRomero0613, great.

**Comment by jessegreenberg** [comment]
I added both to the document.
